test/functional/lib/ipfs-request-dnslink.test.js

```diff
@@ -27,6 +27,7 @@ describe('modifyRequest processing of DNSLinks', function () {
   before(function () {
     global.URL = URL
     global.browser = browser
+    browser.runtime.id = 'testid'
```

Comment:
We're using the same id for `browser.runtime.id` in multiple tests. What is `browser.runtime.id` used for, and would it aid in test debugging to provide a different id?

Can we set the id to `this.testName` or similar?